core/src/main/kotlin/com/github/quillraven/quillycrawler/ashley/system/InteractSystem.kt

```diff
@@ -79,11 +82,13 @@ class InteractSystem(
     when (entity.actionableCmp.type) {
       ActionType.EXIT -> {
         player.add(engine.createComponent(GoToNextLevelComponent::class.java))
+        audioService.playSound(SoundAssets.DROP.descriptor.fileName)
```

Comment:
Perhaps a utility method in `SoundAssets` could shorten this a little bit, since you're using these as identifiers.

Reply:
I have that in different areas already because most methods seem to take a string or a file but they never take a descriptor unfortunately. 
Utility or extension methods make sense especially for Sounds. Will add that as well :)

Reply:
`AssetStorage` supports fetching assets by their descriptor, so it's a matter of changing the `AudioService` if you'd prefer to use that to identify the assets.